modelling_earth/io.py

```diff
@@ -326,3 +326,34 @@ def read_swarm(path):
     # Create a dictionary
     swarm = {"time": time, "step": steps, "positions": positions}
     return swarm
+
+
+def save_swarm(swarm, name, save_path):
+    """
+    Save the particle position as a `hdf` file for each time step.
+
+    Parameters:
+    -----------
+    swarm : dict
+        Dictionary with the time, step and the particles positions.
+        The ``time`` and ``step`` are numpy arrays. ``time`` contains the time of
+        each step in Ma linked to the index of the ``positions`` list.
+        ``positions`` is a list of :class:`pandas.DataFrame` which contains the
+        coordinates `x`, `y` and `z` (in meters) and the flag `cc0` for each time step.
+    name : str
+        Name to save the data.
+    save_path : str or None
+        Path to the folder to save the particle position.
+    """
+    # Define time steps and the particle positions
+    steps = swarm["step"]
+    positions = swarm["positions"]
+    # Get max number of digits on steps
+    number_of_digits = len(str(steps.max()))
+    # Loop to save the particle position for each time step
+    for index in range(len(positions)):
+        filename = "{}_{}.h5".format(name, str(steps[index]).zfill(number_of_digits))
+        positions[index].to_hdf(
+            os.path.join(save_path, filename), key="pd", mode="w", format="fixed"
+        )
+    print("All particle positions have been successfully saved on{}".format(save_path))

```

Comment:
I don't think we should add print statements on these kind of functions. Nevertheless, if you want to add it, we could add a `verbose` bool argument, so in case it's True, the message will be run.
Something like:
```suggestion
    if verbose:
        print("All particle positions have been successfully saved on '{}'".format(save_path))
```